game/view/gameplay/actionhud/hand/cardinfo.lua

```diff
@@ -31,7 +31,7 @@ function CardInfo:init(route)
   local w, h = VIEWDEFS.VIEWPORT_DIMENSIONS()
   self.route = route
   self.card = nil
-  self.position = vec2(w * 0.02, h * 0.07)
+  self.position = vec2(w * 0.02, h * 0.12)
```

Comment:
nevermind more magical values, but why is this one called `position`, while other position members in other classes are named `pos`?

Reply:
Because we are inconsistent. Can this be solved with luafmt? These are field names, not local variable names, so I would be surprised if it was capable of doing so.

Either way, there is now issue #1462 to handle this.